src/index.tsx

```diff
@@ -1,6 +1,5 @@
-import ScriptLoader from "./ScriptLoader";
-export { ScriptLoaderProps } from "./ScriptLoader";
+export { default as default } from "./ScriptLoader";
+export * from "./ScriptLoader";
```

Comment:
WIth `export *` do we still need the `{default as deefault}`?

Reply:
We do need to explicitly declare the default export. Without it, things blow up downstream.